### ISSUE TITLE
Implement download functionality for factsheets

### ISSUE THREAD

**Issue by han-f** [OpenEnergyPlatform/oeplatform#886]
For some users of the OEP (e.g. quantitists and qualitists) it may be helpful to be able to download factsheets and read offline. Please implement a download functionality if this is possible.

For example, could a study factsheet be rendered into a PDF file?

**Comment by Ludee** [comment]
The "old" factsheets can be downloaded in total as CSV.
It makes sense to add the function to generate a formatted document from a single factsheet!

**Comment by adelmemariani** [comment]
Yes, the factsheets can be downloadable as JSON and PDF files (or CSV if needed). However, the full functionality of factsheets belongs to the future work packages - according to our updated timeline. I will keep this requirement in mind. Also, the new (not finished yet) RDF-based factsheets are not only downloadable but also 'shareable' in the sense that we can export a factsheet, send it by email, import a factsheet, modify it, and ... Also we could think about a 'share' button that shares a factsheet among selected users.
